### PR TITLE
Make compliant with cmd2 > 0.10.0

### PR DESCRIPTION
**Pull request by zegervdv**
Fix for #3

### CODE REVIEW

File: cmd2_abbrev/abbrev.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def __init__(self, *args, **kwargs):
         # code placed here runs after cmd2 initializes
         # this is where you register any hook functions
         self.abbrev = False
-        self.settable.update({'abbrev': 'Accept command abbreviations'})
+        self.settables.update({'abbrev': 'Accept command abbreviations'})
```
Comment (tleonhardt) [inline cmd2_abbrev/abbrev.py]: There is a new `add_settable` method and `Settable` class which should be used like so:
```Python
self.add_settable(cmd2.Settable('abbrev', bool, 'Accept command abbreviations'))
```

**Review by tleonhardt — CHANGES_REQUESTED** [review]
Hi @zegervdv,

Thanks a lot for the PR!  Sometimes we forget about this plugin when updating `cmd2`.  

Can you please make the change suggested?

File: cmd2_abbrev/abbrev.py (hunk 1 above)
Comment (zegervdv) [reply cmd2_abbrev/abbrev.py]: sure, fixed it.

It is a really useful plugin though :)

**Review by tleonhardt — APPROVED** [review]
Looks good

**Comment by tleonhardt** [comment]
I'll put out a new release sometime tonight or this weekend.

**Pull request merged**